express_cut/express_service/serializers.py

```diff
@@ -100,13 +100,19 @@ def validate_schedule(timeslots, stylist, date, update = False):
         elif not update and schedule:
             raise ValidationError("Error validating timeslot times. Can not create schedule if it already exists")
 
-        for timeslot in timeslots:
-            time = dict(timeslot)
+        # This should be ok even if its O(n^2) since not a lot of timeslots should be on each schedule.
+        for i in range(len(timeslots)):
```

Comment:
It looks like the logic that's trying to be implemented here may benefit in readability from using the enumerate(list) function. Here's a link to a description of it: https://note.nkmk.me/en/python-enumerate-start/#:~:text=Tags%3A%20Python%2C%20List-,enumerate()%20in%20Python,built%2Din%20function%20enumerate()%20.

Reply:
Ill look into it and try to make the change

Reply:
Ok made the changes to use enumerate